Tools/Google.Apis.Release/Google.Apis.Release.csproj

```diff
@@ -55,8 +55,8 @@
     <Compile Include="Program.cs" />
     <Compile Include="ProjectExtenstions.cs" />
     <Compile Include="Properties\AssemblyInfo.cs" />
+    <Compile Include="Repositories\Git.cs" />
     <Compile Include="Repositories\Hg.cs" />

```

Comment:
I think I would leave it for now, no harm... I know that github might work with Hg in the future, so why not include it as well...